src/databricks/azext_databricks/custom.py

```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
+
+
+from azure.cli.core.util import sdk_no_wait
+
+
+def create_databricks_workspace(cmd, client,
+                                resource_group_name,
+                                workspace_name,
+                                location,
+                                sku_name,
+                                managed_resource_group=None,
+                                aml_workspace_id=None,
+                                custom_virtual_network_id=None,
+                                custom_public_subnet_name=None,
+                                custom_private_subnet_name=None,
+                                enable_no_public_ip=False,
+                                load_balancer_backend_pool_name=None,
+                                load_balancer_id=None,
+                                relay_namespace_name=None,
+                                storage_account_name=None,
+                                storage_account_sku_name=None,
+                                vnet_address_prefix=None,
+                                tags=None,
+                                no_wait=False):
+    body = {}
+    body['tags'] = tags  # dictionary
+    body['location'] = location  # str

```

Comment:
can we get default location from resource group if emit ?

Reply:
added get_default_location_from_resource_group